src/components/editMeasure/measureDetails/MeasureDetailsSidebar.tsx

```diff
@@ -47,6 +47,17 @@ export default function MeasureDetailsSidebar(
 
   useEffect(() => {
     if (links) setMeasureGroups(links);
+
+    if (!measure?.groups?.length) {
+      const baseURL = "/measures/" + measure?.id + "/edit/measure-groups";
+      setMeasureGroups([
+        {
+          title: "MEASURE GROUP 1",
+          href: `${baseURL}`,
+          dataTestId: "leftPanelMeasureInformation-MeasureGroup1",
+        },
+      ]);
+    }
   }, [measure?.groups]);
```

Comment:
This may not be a part of your story, but why does the component name say MeasureDetailsSidebar?
Can we rename it ? or is it a shared side bar for both measureDetails and measureGroups ?

Reply:
Yeah it is shared between both and yeah I'll change the name